content/help/ops/setup/validation/index.md

```diff
@@ -0,0 +1,321 @@
+      # webhook, e.g. istio-galley.istio-system.svc:443
+      name: istio-galley
+      namespace: istio-system
+      path: /admitmixer
+  failurePolicy: Fail
+  name: mixer.validation.istio.io
+  namespaceSelector: {}
+  rules:
+  - apiGroups:
+    - config.istio.io
+    apiVersions:
+    - v1alpha2
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - rules
+    - attributemanifests
+    - circonuses
+    - deniers
+    - fluentds
+    - kubernetesenvs
+    - listcheckers
+    - memquotas
+    - noops
+    - opas
+    - prometheuses
+    - rbacs
+    - servicecontrols
+    - solarwindses
+    - stackdrivers
+    - statsds
+    - stdios
+    - apikeys
+    - authorizations
+    - checknothings
+    - listentries
+    - logentries
+    - metrics
+    - quotas
+    - reportnothings
+    - servicecontrolreports
+    - tracespans
+{{< /text >}}
+
+If the `validatingwebhookconfiguration` doesn’t exist, verify the
+`istio-galley-configuration` `configmap` exists. `istio-galley` uses
+the data from this configmap to create and update the
+`validatingwebhookconfiguration`.
+
+{{< text bash yaml >}}
+$ kubectl -n istio-system get configmap istio-galley-configuration -o jsonpath='{.data}'
+map[validatingwebhookconfiguration.yaml:apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istio-galley
+  namespace: istio-system
+  labels:
+    app: istio-galley
+    chart: galley-1.0.0
+    release: istio
+    heritage: Tiller
+webhooks:
+  - name: pilot.validation.istio.io
+    clientConfig:
+      service:
+        name: istio-galley
+        namespace: istio-system
+        path: "/admitpilot"
+      caBundle: ""
+    rules:
+      - operations:
+      (... snip ...)
+  {{< /text >}}
+
+If the webhook array in `istio-galley-configuration` is empty and
+you're using `helm template` or `helm install`, verify `--set
+galley.enabled` and `--set global.configValidation=true` options are
+set. If you're not using helm, you'll need to find a generate
+YAML that includes the populated webhook array.
+
+The istio-galley validation configuration is fail closed. If
+configuration exists and is scoped properly, the webhook will be
+invoked. A missing `caBundle`, bad cert, or network connectivity
+problem will produce an error message when the resource is
+created/updated. If you don’t see any error message and the webhook
+wasn’t invoked and the webhook configuration is valid, your cluster is
+misconfigured.
+
+## Creating configuration fails with x509 certificate errors
+
+`x509: certificate signed by unknown authority` related errors are
+typically caused by an empty `caBundle` in the webhook
+configuration. Verify that it is non-empty (see "verify webhook
+configuration" above). The `istio-galley` deployment consciously
+reconciles webhook configuration used the `istio-galley-configuration`
+`configmap` and CA certificate mounted from
```

Comment:
Shouldl CA be Citadel?

Reply:
No, this is the CA root certificate. I replaced the wording to just be "root certificate".